hooks/publish.py

```diff
@@ -257,7 +257,7 @@ def init_task_settings(self, item):
                 # Add in the version key if applicable
                 setting.extra["fields"]["version"] = \
                     self.TemplateSettingWidget.TemplateField(
-                        "version", publish_version, "str", editable=False, is_missing=False)
+                        "version", publish_version, "str", editable=True, is_missing=False)
```

Comment:
The default should be non-editable.

Reply:
If this is left non-editable, users won't be able to change the default "1" which gets assigned if `item.properties.fields["version"]` doesn't exist. That's why switched it back to editable. Do you want me to limit the version changes only via `item.properties.fields` and remove it from task settings widget?

Reply:
Editing "version" is very ingest specific, in the vast majority of cases you never want the user to be mucking with the version value, so the default in the base hook should be non-editable. Feel free to change that default for ingest, but since I assume you want the same version value to propagate to all Templates, it seems like it would be better to expose version via the item properties, where it will get propagated to all template fields.